### PR TITLE
Assorted CSS tweaks

### PR DESCRIPTION
**Pull request by XhmikosR**
@emedvedev this is untested and I want to clean up/squash the patches later, but can you deploy this so that we preview it?

TODO:

- [x] the button's font-family isn't inherited (so it's not the same as the other elements)
- [ ] maybe make font-size bigger

### CODE REVIEW

**Comment by XhmikosR** [comment]
BTW @MartijnCuppens you are welcome to play with this repo; it's what we use for our Slack :)

**Comment by MartijnCuppens** [comment]
Ok, I'll have a look when previews are available.

**Comment by emedvedev** [comment]
Deployed!

File: scss/_common.scss [hunk 1]
```diff
@@ -6,6 +6,7 @@ body {
 }
 
 html {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif;
```
Comment (MartijnCuppens) [inline scss/_common.scss]: `system-ui` is the standerdized (working Draft) keyword:
```suggestion
  font-family: system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif;
```
see https://caniuse.com/#feat=font-family-system-ui

Comment (XhmikosR) [reply scss/_common.scss]: I just used what we use in Bootstrap, minus the Emoji fonts.

Comment (MartijnCuppens) [reply scss/_common.scss]: nvm, not sure if https://infinnie.github.io/blog/2017/systemui.html still applies

**Comment by emedvedev** [comment]
@XhmikosR are you still planning to work on this? If you don't have the time, I can merge what's already here if it's done and tested.

**Comment by XhmikosR** [comment]
@emedvedev it works but I'd like to fix the button situation too. Let me try to fix that today.

**Comment by XhmikosR** [comment]
Should be OK now, but better not squash the patches :)

**Pull request merged**